npm-shrinkwrap.json

```diff
@@ -1,1109 +0,0 @@
-{
-  "name": "codeclimate-eslint",
-  "version": "0.0.3",
-  "dependencies": {
-    "acorn": {
-      "version": "3.3.0",
```

Comment:
Hmmm. Yeah, I don't know what happened there. I just used our `make shrinkwrap` rule. I'll look into it, thanks for noticing.